packages/web/src/components/page/Page.js

```diff
@@ -20,7 +20,7 @@ const HEADER_MARGIN_PX = 32
 const MIN_GUTTER_WIDTH = 20
 
 // Responsible for positioning the header
-const HeaderContainer = ({ header, containerRef }) => {
+const HeaderContainer = ({ header, containerRef, useSearch }) => {
```

Comment:
very nit: useSearch sounds like a hook, but it doesn't look like it is? maybe `shouldUseSearch`